### ISSUE TITLE
No Freeze for gradients during training

### ISSUE THREAD

**Issue by inspirit** [nii-yamagishilab/Capsule-Forensics-v2#7]
Hello, 
I have noticed that you do not freeze gradients when training capsules (default value for train is always False), I would expect we dont want gradient for VGG layers when training capsules?
https://github.com/nii-yamagishilab/Capsule-Forensics-v2/blob/90a6ad2ebe4af3f85022e18ba77940988011d379/model_big.py#L46-L50

**Comment by honghuy127** [comment]
In this update version, gradient from layer 0 to 8 are frozen and from layer 9 to 17 are enable. It may help in some cases.

**Comment by honghuy127** [comment]
However, I have not updated the source code for training CapsNet and finetuning VGG extractor yet.
